frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/AuthServingRuntimeSection.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import {
+  Alert,
+  Button,
+  Checkbox,
+  FormGroup,
+  FormSection,
+  Popover,
+  Stack,
+  StackItem,
+  getUniqueId,
+} from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import { UpdateObjectAtPropAndValue } from '~/pages/projects/types';
+import {
+  CreatingInferenceServiceObject,
+  CreatingServingRuntimeObject,
+} from '~/pages/modelServing/screens/types';
+
+import ServingRuntimeTokenSection from './ServingRuntimeTokenSection';
+
+type AuthServingRuntimeSectionProps = {
+  data: CreatingServingRuntimeObject | CreatingInferenceServiceObject;
+  setData:
+    | UpdateObjectAtPropAndValue<CreatingServingRuntimeObject>
+    | UpdateObjectAtPropAndValue<CreatingInferenceServiceObject>;
+  allowCreate: boolean;
+  publicRoute?: boolean;
+};
+
+const AuthServingRuntimeSection: React.FC<AuthServingRuntimeSectionProps> = ({
+  data,
+  setData,
+  allowCreate,
+  publicRoute,
+}) => {
+  const createNewToken = React.useCallback(() => {
+    const name = 'default-name';
+    const duplicated = data.tokens.filter((token) => token.name === name);
+    const duplicatedError = duplicated.length > 0 ? 'Duplicates are invalid' : '';
+    setData('tokens', [
+      ...data.tokens,
+      {
+        name,
+        uuid: getUniqueId('ml'),
+        error: duplicatedError,
+      },
+    ]);
+  }, [data.tokens, setData]);
+
+  return (
+    <Stack>
+      {!allowCreate && (
+        <StackItem>
+          <Popover
+            showClose
+            bodyContent={
+              publicRoute
+                ? 'Model route and token authorization can only be changed by administrator users.'
+                : 'Token authorization can only be changed by administrator users and component Authorino needs to be installed.'
+            }
+          >
+            <Button variant="link" icon={<HelpIcon />} isInline>
```

Comment:
We should be using `OutlinedQuestionCircleIcon`. See https://www.patternfly.org/design-foundations/icons